dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/settings/SettingsServiceBean.java

```diff
@@ -430,7 +430,13 @@ Whether Harvesting (OAI) service is enabled
         HandleNetAdmPrivPhrase,
         HandleNetIndex,
 
-        TimerServer;
+        TimerServer,
+
+        /**
+         * The number of minutes for which a password reset token is valid .
```

Comment:
I think the name is meaningful enough that the comment is not necessary. We should only use javadocs when it is really necessary and the name doesn't give away the full meaning.